Mobile/Android/Templates/Release_2.5.0/xxxx-oauth2.md

```diff
@@ -0,0 +1,54 @@
+###  OAuth2 
+
+#### Pr: https://github.com/owncloud/android/pull/XXX 
+
+
+---
+
+ 
+| TestID | Test Case | Steps | Expected Result | Result | Related Comment |
+| :----: | :-------- | :---- | :-------------- | :----: | :-------------- |
+|**Login View**|||||||
+| 1 | Initial look | Open the app. Check both orientations | Correct view. Only URL and connect button are shown  |  |  |
+| 2 | OAuth2 URL | 1. Set an URL of a server with OAuth2<br>2. Tap on connect.<br> Check both orientations | Redirected to webview. No more options shown |  |  |
+| 3 | Basic Auth URL | 1. Set an URL of a server without OAuth2<br>2. Tap on connect.<br>Check both orientations | Fields username and password are shown |  |  |
+| 4 | Redirected URL | 1. Set a redirection URL <br>2. Tap on connect. <br>Check both orientations | Fields username and password are shown  |  |
+| 5 | SAML URL | Set an SAML URL (not supported by the app)<br>Check both orientations | Correct error messaege |  |  |
+| 6 | Wrong URL | Set an incorrect URL. <br>Check both orientations | Correct error messaege |  |  |
+| 7 | Wrong Authorizathion endpoint | 1. In customization file, change auth endpoint for a incorrect one<br>2. Enter OAuth2 URL and connect. <br>Check both orientations | Correct error handling |  |  |
+|**Branding**|||||||
+| 8 | OAuth2 URL hardcoded | Open the app.<br> Check both orientations | Correct view |  |  |
+| 9 | OAuth2 URL hardcoded and hidden | Open the app.<br> Check both orientations | Correct view |  |  |
+| 10 | Help link hidden | Open the app.<br> Check both orientations | Correct view |  |
+|**OAuth2 UI flow**|||||||
+| 11 | Webview | Enter OAuth2 URL and connect.<br> Check both orientations | Correct webview  |  |  |
+| 12 | Correct credentials | 1. Enter OAuth2 URL and connect.<br>2. Enter correct credentials. <br>Check both orientations | 1. Webview is opened<br> 2. Credentials are checked and webview is closed  |  |  |
+| 13 | Token expired | Wait until token expires| User is redirected to the webview to enter again the credentials |  |  |
+| 14 | Wrong credentials | 1. Enter OAuth2 URL and connect.<br>2. Enter wrong credentials. <br>Check both orientations | 1. Webview is opened<br> 2. Webview can be closed  |  |  |
```

Comment:
TC 11 is focused in functionality. Previous TCs are focused in UI. Do you mean this?